src/accounts/AccountPage.tsx

```diff
@@ -105,12 +99,6 @@ const AccountAboutPage: FC = () => {
             text="Save"
           />
         </FlexBox>
-        {allowSelfRemoval && showLeaveOrgButton && (
-          <>
-            <hr className="account-settings--divider" />
-            <LeaveOrgButton />
-          </>
-        )}
         {shouldShowDeleteFreeAccountButton && <DeleteFreeAccountButton />}
```

Comment:
The "Leave Org" button was only here for backwards compatibility for situations where the feature flag was turned off. It now lives on the org settings page.